.github/workflows/main.yml

```diff
@@ -65,7 +65,7 @@ jobs:
       - name: Build
         run: |
           mkdir build
-          cmake -S . -B build -DLLVM_INCLUDE_TESTS=On
+          cmake -S . -B build -DLLVM_INCLUDE_TESTS=On -DLLVM_DIR=/usr/lib/llvm-16/cmake

```

Comment:
Somehow this is in `/usr/local/opt/llvm@16/lib/cmake` on macOS, but apparently cmake doesn't notice this error